.gitignore

```diff
@@ -2,6 +2,7 @@
 /.sass-cache/
 dist
 node_modules/
+.idea/
```

Comment:
This shouldn't be here. Because it's unrelated to this PR but mainly because it deserves in your global gitignore.